
This patch changes font color used in texts with running context from green to amber


diff --speed-large-files --no-dereference --minimal -Naur Monochrome-8569391fd35dc01aaec19f8085b3002ede8148b2/base.xml Monochrome-8569391fd35dc01aaec19f8085b3002ede8148b2/base.xml
--- Monochrome-8569391fd35dc01aaec19f8085b3002ede8148b2/base.xml	2022-04-26 01:09:17.000000000 +0200
+++ Monochrome-8569391fd35dc01aaec19f8085b3002ede8148b2/base.xml	2022-01-20 14:30:14.404299365 +0100
@@ -36,6 +36,9 @@
     <fontdef name="basefont_l_off" from="basefont_l">
         <color>#7F7F7F</color>
     </fontdef>
+    <fontdef name="basefont_l_running" from="basefont_l">
+        <color>#E0991A</color>
+    </fontdef>
     <fontdef name="basefont_m_good" from="basefont_m">
         <color>#00FF00</color>
     </fontdef>
@@ -48,6 +51,9 @@
     <fontdef name="basefont_m_off" from="basefont_m">
         <color>#7F7F7F</color>
     </fontdef>
+    <fontdef name="basefont_m_running" from="basefont_m">
+        <color>#E0991A</color>
+    </fontdef>
     <fontdef name="basefont_xs_good" from="basefont_xs">
         <color>#00FF00</color>
     </fontdef>
@@ -57,6 +63,9 @@
     <fontdef name="basefont_xs_warn" from="basefont_xs">
         <color>#FFA500</color>
     </fontdef>
+    <fontdef name="basefont_xs_running" from="basefont_xs">
+        <color>#E0991A</color>
+    </fontdef>
 
     <!-- textarea with small, medium, large and extra large font -->
     <textarea name="basetext">
